pages/pil/read/index.js

```diff
@@ -65,6 +65,7 @@ module.exports = settings => {
     res.locals.static.profile = req.profile;
     res.locals.static.currentPath = req.originalUrl;
     res.locals.static.isLicenceHolder = req.user.profile.id === req.profileId;
+    res.locals.static.isAsruUser = !!req.user.profile.asruUser;
```

Comment:
This data point is already available on the client at `static.profile.asruUser` so it's better to use that rather than add _yet another data point_.

Reply:
Yeah `req.profile` in this case is the PIL holder, not the currently logged in user.

I thought it was as well, but both `isAsru` and `isAsruUser` were undefined until I added this.

Reply:
See my eventual edit above.